crates/primitives/src/transaction/access_list.rs

```diff
@@ -51,6 +54,20 @@ pub struct AccessList(
     pub Vec<AccessListItem>,
 );
 
+impl Deref for AccessList {
+    type Target = Vec<AccessListItem>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for AccessList {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.0
+    }
+}
```

Comment:
pedantic nit:
this should go below the `impl AccessList` block

Reply:
done